var/spack/repos/builtin/packages/py-pytecplot/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack import *
+
+
+class PyPytecplot(PythonPackage):
+    """The pytecplot library is a high level API that connects your
+    Python script to the power of the Tecplot 360 visualization engine.
+    It offers line plotting, 2D and 3D surface plots in a variety of formats,
+    and 3D volumetric visualization. Familiarity with Tecplot 360 and the
+    Tecplot 360 macro language is helpful, but not required."""
+
+    homepage = "https://www.tecplot.com/docs/pytecplot/"
+    pypi = "pytecplot/pytecplot-1.4.2.zip"
+
+    version("1.4.2", sha256="586a2ee947314ddd2f28be5523911dd298465b8f6a9145ba351866d5d695ef0d")
+
+    variant("numpy", default=False, description="Add numpy dependency.")
+    variant("ipython", default=False, description="Add ipython dependency.")
```

Comment:
I usually try to keep the Spack variant names the same as pip. When you install with pip, you would use `pip install pytecplot[extras]` to get numpy/ipython. Should we do the same here, a single `+extras` variant?